sklearn/feature_selection/rfe.py

```diff
@@ -425,3 +460,8 @@ def fit(self, X, y):
         # here, the scores are normalized by len(cv)
         self.grid_scores_ = scores / len(cv)
         return self
+
+    @staticmethod
+    def _step_score_on_test(X_test, y_test, scorer):
```

Comment:
I'm not sure it is worht adding this function, why not just doing the lambda inline?